html/pfappserver/root/static/admin/config/items.js

```diff
@@ -45,6 +45,7 @@ Items.prototype.post = function(options) {
  * The ItemView class defines the DOM operations from the Web interface.
  */
 var ItemView = function(options) {
+    var that = this;
```

Comment:
You're right but I usually like to have it always declared so it can always be used